Packages/com.unity.inputsystem/InputSystem/Editor/AssetImporter/InputActionImporter.cs

```diff
@@ -331,38 +319,72 @@ public static void CreateInputAsset()
                 InputActionAsset.kDefaultAssetLayoutJson, InputActionAssetIconLoader.LoadAssetIcon());
         }
 
-        // When an action asset is renamed, copied, or moved in the Editor, the "Name" field in the JSON will
-        // hold the old name and won't match what's in memory (asset looks "dirty"). To work around this, we
-        // must flush the updated JSON to the file whenever this operation occurs.
+        // File extension of the associated asset
+        private const string kFileExtension = "." + InputActionAsset.Extension;
```

Comment:
Seems reasonable adding null check but the extra movement in the file made it harder to diff.